jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/Rabit.java

```diff
@@ -92,4 +131,73 @@ public static int getWorldSize() throws XGBoostError {
     checkCall(XGBoostJNI.RabitGetWorldSize(out));
     return out[0];
   }
+
+  public static float[] allReduce(float[] elements, OpType op) {
```

Comment:
I think you can left only one of them and comment something like purely for test?